superset-frontend/src/datasource/DatasourceEditor.jsx

```diff
@@ -516,6 +516,25 @@ class DatasourceEditor extends React.PureComponent {
             control={<TextControl controlId="fetch_values_predicate" />}
           />
         )}
+        {this.state.isSqla && (
+          <Field
+            fieldKey="extra"
+            label={t('Extra')}
+            description={t(
+              'Extra data to specify table metadata. Currently supports ' +
+                'certification data of the format: `{ "certification": { "certified_by": ' +
+                '"Taylor Swift", "details": "This table is the source of truth." ' +
```

Comment:
Lol, I like how we codified `Taylor Swift` to the actual user interface. She's certainly not a controversial figure, but should we use a more "business friendly" example? E.g. `"certified_by": "Data Platform Team"`

Reply:
+1 on Taylor Swift
<img width="931" alt="Screen Shot 2020-10-31 at 1 44 50 PM" src="https://user-images.githubusercontent.com/487433/97789732-f16a3c80-1b7f-11eb-8adf-f8597e7a1e3b.png">

Or Batman. Batman may work too.

Reply:
Actually, let's draw the line on examples (where random/funny is ok) and keep tooltips / descriptions more businessy